e2e/wdio/serialization/install_sitecore_cli.ps1

```diff
@@ -0,0 +1,14 @@
+dotnet new tool-manifest
+dotnet nuget add source -n Sitecore https://sitecore.myget.org/F/sc-packages/api/v3/index.json
+dotnet tool install Sitecore.CLI
```

Comment:
Can the `.config` and `.sitecore` files be checked in instead, so that only `dotnet tool restore` is required? Similar to our setup in other repos.